modules/002-deckhouse/templates/deployment.yaml

```diff
@@ -213,6 +213,8 @@ spec:
               value: /tmp/.bash_history
             - name: DEBUG_HTTP_SERVER_ADDR
               value: "127.0.0.1:9652"
+            - name: CRYPTOGRAPHY_OPENSSL_NO_LEGACY
+              value: "1"
```

Comment:
not sure about this env, does it really have to do something with the deckhouse container?

Reply:
I'm not sure, but I took it from here https://github.com/deckhouse/deckhouse/commit/06b216188113b6e7d586d17743277cae6f303a10. @RomanenkoDenys Is this important?

Reply:
According to the documentation, this variable disables the use of legacy cryptographic algorithms. https://cryptography.io/en/latest/openssl/#legacy-provider-in-openssl-3-x

Reply:
I see, now it makes sense, thanks